src/System.Text.Json/src/System/Text/Json/Node/JsonArray.cs

```diff
@@ -0,0 +1,675 @@
+        /// </summary>
+        /// <param name="item">Item to find.</param>
+        /// <returns>The zero-based starting index of the search. 0 (zero) is valid in an empty list.</returns>
+        public int LastIndexOf(JsonNode item) => _list.LastIndexOf(item);
+
+        /// <summary>
+        ///   Removes all elements from the JSON array.
+        /// </summary>
+        public void Clear() => _list.Clear();
+
+        /// <summary>
+        ///   Removes the first occurrence of a specific object from the JSON array.
+        /// </summary>
+        /// <param name="item">
+        ///   The object to remove from the JSON array. The value can be null and it represents null JSON node.
+        /// </param>
+        /// <returns>
+        ///   <see langword="true"/> if the item is successfully found in a JSON array and removed,
+        ///   <see langword="false"/> otherwise.
+        /// </returns>
+        public bool Remove(JsonNode item) => _list.Remove(item);
+
+        /// <summary>
+        ///   Removes all the elements that match the conditions defined by the specified predicate.
+        /// </summary>
+        /// <param name="match">
+        ///   Thepredicate delegate that defines the conditions of the elements to remove.
+        /// </param>
+        /// <returns>The number of elements removed from the JSOJN array.</returns>
+        public int RemoveAll(Predicate<JsonNode> match) => _list.RemoveAll(match);
+
+        /// <summary>
+        ///   Removes the item at the specified index of the JSON array.
+        /// </summary>
+        /// <param name="index">
+        ///   The zero-based index of the element to remove.
+        /// </param>
+        public void RemoveAt(int index) => _list.RemoveAt(index);
+
+        /// <summary>
+        ///   Copies the JSON array or a portion of it to an array.
+        /// </summary>
+        /// <param name="array">
+        ///   The one-dimensional Array that is the destination of the elements copied from JSON array.
+        ///   The Array must have zero-based indexing.
+        /// </param>
+        /// <param name="arrayIndex">The zero-based index in array at which copying begins.</param>
+        void ICollection<JsonNode>.CopyTo(JsonNode[] array, int arrayIndex) => _list.CopyTo(array, arrayIndex);
+
+        /// <summary>
+        ///   Returns an enumerator that iterates through the JSON array values.
+        /// </summary>
+        /// <returns>An enumerator structure for the <see cref="JsonArray"/>.</returns>
+        IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
+
+        /// <summary>
+        ///   Returns an enumerator that iterates through the JSON array values.
+        /// </summary>
+        /// <returns>An enumerator structure for the <see cref="JsonArray"/>.</returns>
+        public IEnumerator<JsonNode> GetEnumerator() => new JsonArrayEnumerator(this);
```

Comment:
```suggestion
        public JsonArrayEnumerator GetEnumerator() => new JsonArrayEnumerator(this);
```

That might also require an explicit interface implementation for one returning `IEnumerator<JsonNode>`.  I don't remember.

Reply:
Ok, I will check it.